test/common.py

```diff
@@ -949,9 +948,9 @@ def require_jspi(self):
       return
 
     if 'EMTEST_SKIP_JSPI' in os.environ:
-      self.skipTest('test requires node >= 19 or d8 (and EMTEST_SKIP_JSPI is set)')
+      self.skipTest('test requires node canary or d8 (and EMTEST_SKIP_JSPI is set)')
```

Comment:
While you're at it, can you fix this for exnref too?
https://github.com/emscripten-core/emscripten/blob/7e796337c1d04ac7e95295ef329e10ad2c45512b/test/common.py#L905-L919